src/setup-docker/templates/fabric-docker/snapshot-scripts.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+__getSnapshotNodes() {
```

Comment:
1. do we need `__` prefix ? :)
2. we used to write `function blah() {` why convention has changed here?